setuptools/build_meta.py

```diff
@@ -385,6 +385,8 @@ def _build_with_temp_dir(
                 "--dist-dir",
                 tmp_dist_dir,
             ]
+            if arbitrary_args:
+                sys.argv += self._arbitrary_args(config_settings)
```

Comment:
We probably can avoid the conditional all together this way:

```suggestion
                tmp_dist_dir,
                *arbitrary_args,
            ]
```

(untested suggestion)

Reply:
applied as suggested